.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
```

Comment:
Don't you publish Test PyPI for the test sake? I'm trying to do so in pytd: https://github.com/treasure-data/pytd/pull/75/commits/7dce66e473cd6850801b022c8bf4b8909446d949#diff-8f47ac56283c70f1b8a41d586871524bR20-R26

Reply:
Personally, I think it's sufficient to deploy to Test PyPI from the local environment if needed. Test PyPI is good to test publishing wheels but it's not a good place to evaluate installation with dependencies since not all Python packages are uploaded on it.